Machine Learning/Business Metrics Recipes/twitter_trends/data/tweets.csv

```diff
@@ -66,7 +66,7 @@
-{"created_at": "Fri Jun 23 15:26:45 +0000 2017", "id": 878273126471217152, "id_str": "878273126471217152", "text": "SIM: a compra de um bilhete de trem pode ser uma tarefa complexa com margem \u00e0 v\u00e1rios problemas!\u2026 https://t.co/SxYCpB2jtJ", "truncated": true, "entities": {"hashtags": [], "symbols": [], "user_mentions": [], "urls": [{"url": "https://t.co/SxYCpB2jtJ", "expanded_url": "https://twitter.com/i/web/status/878273126471217152", "display_url": "twitter.com/i/web/status/8\u2026", "indices": [97, 120]}]}, "metadata": {"iso_language_code": "pt", "result_type": "recent"}, "source": "<a href=\"http://twitter.com\" rel=\"nofollow\">Twitter Web Client</a>", "in_reply_to_status_id": null, "in_reply_to_status_id_str": null, "in_reply_to_user_id": null, "in_reply_to_user_id_str": null, "in_reply_to_screen_name": null, "user": {"id": 3287881129, "id_str": "3287881129", "name": "Dynatrace Latam", "screen_name": "Dynatrace_Latam", "location": "S\u00e3o Paulo, Brasil", "description": "P\u00e1gina de APM da Dynatrace: novidades sobre as melhores pr\u00e1ticas para otimizar o desempenho de aplica\u00e7\u00f5es web.", "url": "http://t.co/s7ykiZTihS", "entities": {"url": {"urls": [{"url": "http://t.co/s7ykiZTihS", "expanded_url": "http://www.dynatrace.com", "display_url": "dynatrace.com", "indices": [0, 22]}]}, "description": {"urls": []}}, "protected": false, "followers_count": 28, "friends_count": 40, "listed_count": 13, "created_at": "Wed Jul 22 18:45:04 +0000 2015", "favourites_count": 10, "utc_offset": -10800, "time_zone": "Brasilia", "geo_enabled": false, "verified": false, "statuses_count": 347, "lang": "pt", "contributors_enabled": false, "is_translator": false, "is_translation_enabled": false, "profile_background_color": "C0DEED", "profile_background_image_url": "http://abs.twimg.com/images/themes/theme1/bg.png", "profile_background_image_url_https": "https://abs.twimg.com/images/themes/theme1/bg.png", "profile_background_tile": false, "profile_image_url": "http://pbs.twimg.com/profile_images/712289342094503936/z9ug1jd3_normal.jpg", "profile_image_url_https": "https://pbs.twimg.com/profile_images/712289342094503936/z9ug1jd3_normal.jpg", "profile_banner_url": "https://pbs.twimg.com/profile_banners/3287881129/1487099052", "profile_link_color": "1DA1F2", "profile_sidebar_border_color": "C0DEED", "profile_sidebar_fill_color": "DDEEF6", "profile_text_color": "333333", "profile_use_background_image": true, "has_extended_profile": false, "default_profile": true, "default_profile_image": false, "following": false, "follow_request_sent": false, "notifications": false, "translator_type": "none"}, "geo": null, "coordinates": null, "place": null, "contributors": null, "is_quote_status": false, "retweet_count": 0, "favorite_count": 0, "favorited": false, "retweeted": false, "possibly_sensitive": false, "lang": "pt"}
```

Comment:
@ypid-geberit Why the change from `ser` to `ser_`?

Reply:
Good catch. I don’t know the reason either so it looks like this has slipped in. I removed that change.